pkg/helmraiser/helm.go

```diff
@@ -0,0 +1,152 @@
+package helmraiser
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+
+	jsonnet "github.com/google/go-jsonnet"
+	"github.com/google/go-jsonnet/ast"
+	"github.com/pkg/errors"
+	yaml "gopkg.in/yaml.v3"
+)
+
+func confToArgs(conf map[string]interface{}) ([]string, []string, error) {
```

Comment:
The format of `conf` is not only implicitly defined based on how this code works.

For documentation purposes, I think it would be better to have a struct:

```go
type Conf struct {
  Values map[string]interface{}
  Flags map[string]interface{}
}
```

To get the data from the `interface{}` you get from jsonnet into the struct, best use the following:

```go
d, err := json.Marshal(data[2])

var conf Conf
err := json.Unmarshal(d, &conf)
```

The overhead of marshaling here can be ignored compared to the time consumed e.g. by the Jsonnet compiler. However, this avoids extensive use of `reflect` and also checks for type integrity (that the passed argument is actually a `map[string]interface{}`, etc. without raising panics (your current code does)